ci/unit/docker-setup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# This is intended to be run the plugin's root directory. `ci/unit/docker-test.sh`
+# Ensure you have Docker installed locally and set the ELASTIC_STACK_VERSION environment variable.
+set -e
+
+VERSION_URL="https://raw.githubusercontent.com/elastic/logstash/master/ci/logstash_releases.json"
+
+if [ "$ELASTIC_STACK_VERSION" ]; then
+    echo "Fetching versions from $VERSION_URL"
+    VERSIONS=$(curl $VERSION_URL)
+    if [[ "$SNAPSHOT" = "true" ]]; then
+      ELASTIC_STACK_RETRIEVED_VERSION=$(echo $VERSIONS | jq '.snapshots."'"$ELASTIC_STACK_VERSION"'"')
+      echo $ELASTIC_STACK_RETRIEVED_VERSION
+    else
+      ELASTIC_STACK_RETRIEVED_VERSION=$(echo $VERSIONS | jq '.releases."'"$ELASTIC_STACK_VERSION"'"')
+    fi
+    if [[ "$ELASTIC_STACK_RETRIEVED_VERSION" != "null" ]]; then
+      # remove starting and trailing double quotes
+      ELASTIC_STACK_RETRIEVED_VERSION="${ELASTIC_STACK_RETRIEVED_VERSION%\"}"
+      ELASTIC_STACK_RETRIEVED_VERSION="${ELASTIC_STACK_RETRIEVED_VERSION#\"}"
+      echo "Translated $ELASTIC_STACK_VERSION to ${ELASTIC_STACK_RETRIEVED_VERSION}"
+      export ELASTIC_STACK_VERSION=$ELASTIC_STACK_RETRIEVED_VERSION
+    fi
+
+    echo "Testing against version: $ELASTIC_STACK_VERSION"
+
+    if [[ "$ELASTIC_STACK_VERSION" = *"-SNAPSHOT" ]]; then
+        cd /tmp
+
+        jq=".build.projects.\"logstash-docker\".packages.\"logstash-$ELASTIC_STACK_VERSION-docker-image.tar.gz\".url"
```

Comment:
It looks like the location of the artifact URL changed recently 🙄

```suggestion
        jq=".build.projects.logstash.packages.\"logstash-$ELASTIC_STACK_VERSION-docker-image.tar.gz\".url"
```

Reply:
-> https://github.com/logstash-plugins/logstash-codec-gzip_lines/pull/10 to trigger CI since I don't have push access to your repo